assets/src/blocks/Gallery/GalleryGrid.js

```diff
@@ -1,5 +1,11 @@
-export const GalleryGrid = ({ images }) => (
-  <div className="container">
+import { useEffect } from '@wordpress/element';
```

Comment:
Maybe this is a good opportunity to look into a better React version for the frontend. Since we only render simple things on the frontend, we could probably do with a very lightweight React implementation. The libs included in WordPress are quite the opposite, and they have a lot of dependencies which are not optimized to reduce bundle size, as they're intended for the post editor.

Reply:
![Screenshot from 2020-10-20 11-12-01](https://user-images.githubusercontent.com/7604138/96565856-314e3b80-12c5-11eb-8646-2d80a50f754b.png)
![Screenshot from 2020-10-20 11-11-46](https://user-images.githubusercontent.com/7604138/96565863-31e6d200-12c5-11eb-9d0d-bf4244a9555d.png)

Reply:
The only code difference between both screenshots is the line enqueuing  `frontendIndex` (and its dependencies) is not executed in the second (I put a check for a query param around it).

Reply:
Not posting this here to say that we shouldn't use an effect for the lightbox. But if we do so we probably want to look into setting up a lightweight standalone React lib for the frontend which doesn't add so many dependencies.

Reply:
@Inwerpsel That's a good point, when we started the move to React we actually thought of using Preact (3kb) in the frontend to minimize the impact, but as at first we didn't ship any React blocks to the frontend (we used Twig instead), I think the idea got buried for a few months. I used Preact a lot in the past and I would say 90% of the times the code is compatible  (sometimes requiring `preact-compat` which is just 2kb), so we could probably do a separate bundle using the same components. I guess it has some risk of not being compatible but maybe it's worth giving it a try. 

https://preactjs.com/guide/v10/differences-to-react

Otherwise I guess we would be fine just using react/reactDOM. 

Apart from all this, for the case of the Gallery, we should probably consider making it a static block...

Reply:
I think we are not going to switch to a different package in this PR so I'll resolve this comment.